agent_code/our_agent_v2/train.py

```diff
@@ -68,6 +73,8 @@ def setup_training(self):
     # Ques to store the transition tuples and coordinate history of agent.
     self.transitions        = deque(maxlen=TRANSITION_HISTORY_SIZE) # long term memory of complete step
     self.coordinate_history = deque([], 10)                         # short term memory of agent position
+    self.n_step_rewards = deque([], N_STEPS)
```

Comment:
Do you have anything planned for this or was it just forgotten? :)

Reply:
```suggestion
```